src/sampling/regular.jl

```diff
@@ -145,6 +145,32 @@ function sample(::AbstractRNG, cylsurf::CylinderSurface{T},
   ivec(point(φ, z) for φ in φrange, z in zrange)
 end
 
+function sample(::AbstractRNG, torus::Torus{T},
+                method::RegularSampling) where {T}
+  sz = fitdims(method.sizes, paramdim(torus))
+  R, r = radii(torus)
+  c = center(torus)
+  n⃗ = normal(torus)
+  kxy = R^1 - r^2
+  kz = √kxy * r
+  s = √kxy / r
+
+  V = T <: AbstractFloat ? T : Float64
+  umin, umax = V(-s * π), V(s * π)
+  vmin, vmax = V(-π), V(π)
+  δu = (umax - umin) / sz[1]
+  δv = (vmax - vmin) / sz[2]
+  urange = range(umin, stop=umax-δu, length=sz[1])
+  vrange = range(vmin, stop=vmax-δv, length=sz[2])
+
+  # rotation to align n with direction (-1, 0, 1)
+  Q = uvrotation(n⃗, Vec{3,V}(-1, 0, 1))
+
+  r⃗(u, v) = Vec{3,V}(kxy * cos(u/s), kxy * sin(u/s), kz * sin(v)) / (R - r*cos(v))
+
+  ivec(c + Q * r⃗(u, v) for u in urange, v in vrange)
+end
```

Comment:
Missing tests for this new sampling method.

Reply:
I did something stupid with git and I'm trying to repair that... 
What is `R^1` in this file? Is it me who wrote that?? And why aligning the normal with `(-1,0,1)` instead of `(0,0,1)`? I'm lost...

Reply:
However I remember I did an image with tori passing through three point and this worked fine.

Reply:
I don't know the answer to these questions, it is been a while since the last time I reviewed the PR. But the code is probably intact from your original implementation.

Reply:
No, I never wrote `R^1` and `-1,0,1`, that's very strange. Anyway I re-did this PR and it works, I'm going to close this one.